src/main/scala/it/pagopa/interop/notifier/server/impl/Main.scala

```diff
@@ -75,7 +72,7 @@ object Main extends App with CORSSupport with Dependencies {
 
       val serverBinding: Future[Http.ServerBinding] = for {
         jwtReader <- getJwtReader()
-        dynamo     = dynamoReader()
+        dynamo     = dynamoNotificationService()
```

Comment:
is it correct to create another instance? see [here](https://github.com/pagopa/interop-be-notifier/pull/22/files?diff=split&w=1#diff-47c83642fe1cfb40057385938304d3336e84ca746a10c7b5d27d46a99a6a817aR63)

Reply:
that's definitively a bug

Reply:
Absolutely a typo. 
I'm going to fix it.